app/models/case_log.rb

```diff
@@ -460,10 +464,69 @@ def self.label_from_value(value)
   end
 
   def self.csv_attributes(user)
-    attributes = attribute_names - %w[owning_organisation_id managing_organisation_id created_by_id] + %w[unittype_sh owning_organisation_name managing_organisation_name created_by_name]
+    attributes = attribute_names + %w[unittype_sh]
+    attributes = replace_csv_id_fields_with_names(attributes)
+
+    attributes = order_csv_attributes(attributes)
+
     user.present? && !user.support? ? attributes - CSV_FIELDS_TO_OMIT : attributes
   end
 
+  def self.replace_csv_id_fields_with_names(initial_attributes)
+    attributes = initial_attributes.clone
```

Comment:
Do we need the `.clone` here?

Reply:
Well it changes the original if we don't do that. I don't know how I feel about that 😅